Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,54 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+    # Running the container with correctly set environment variables, all tasks from run_tests.py should pass
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0, result.returncode)
+
+    # Set incorrect env variable UMOCI_BIN_PATH for ansible playbook extract-operator-bundle.yml - playbook fails to extract
+    def test_negative_extract_operator_bundle(self):
+        exec_cmd = """
+        export UMOCI_BIN_PATH=/root
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        with open("extract-operator-bundle.errormessage") as error_file:
+            if "Result code:" not in error_file.read():
+                print("Result code not found in %s" % error_file)
+                assert False
+        self.assertEqual(55, result.returncode)
```

Comment:
Based on the description this should not be covered by return codes 50-99.
But with "Other Codes" because wrong env variable means either broken default in container image
or wrong overwrite when running the container.

In another words, such test case is exactly "a system or an infrastructure issue"  which should be covered by other return codes based on requirements in the story.

Reply:
Changed to 102.